docs/sphinx/users/comlinetools/conversion.txt

```diff
@@ -15,61 +17,60 @@ To convert a file to single output file (e.g. TIFF):
 The output file format is determined by the extension of the output file, e.g.
 .tiff for TIFF files, .ome.tiff for OME-TIFF, .png for PNG.
 
-All images in the input file are converted by default.  To convert only one
-series:
+.. option:: -series SERIES
 
-::
+    All images in the input file are converted by default.  To convert only one
+    series::
 
-  bfconvert -series 0 /path/to/input output-first-series.tiff
+      bfconvert -series 0 /path/to/input output-first-series.tiff
 
-To convert only one timepoint:
+.. option:: -timepoint TIMEPOINT
 
-::
+    To convert only one timepoint::
 
-  bfconvert -timepoint 0 /path/to/input output-first-timepoint.tiff
+      bfconvert -timepoint 0 /path/to/input output-first-timepoint.tiff
 
-To convert only one channel:
+.. option:: -channel CHANNEL
 
-::
+    To convert only one channel::
 
-  bfconvert -channel 0 /path/to/input output-first-channel.tiff
+      bfconvert -channel 0 /path/to/input output-first-channel.tiff
 
-To convert only one Z section:
+.. option:: -z Z
 
-::
+    To convert only one Z section::
 
-  bfconvert -z 0 /path/to/input output-first-z.tiff
+      bfconvert -z 0 /path/to/input output-first-z.tiff
 
-To convert images between certain indices (inclusive):
+.. option:: -range START END
 
-::
+    To convert images between certain indices (inclusive)::
 
-  bfconvert -range 0 2 /path/to/input output-first-3-images.tiff
+      bfconvert -range 0 2 /path/to/input output-first-3-images.tiff
 
-All images larger than 4096x4096 will be saved as a set of tiles if the output
-format supports doing so.  The default tile size is determined by the input
-format, and can be overridden like this:
+.. option:: -tilex TILEX, -tiley TILEY
```

Comment:
Correct, that may be a Sphinx limitation though which I cannot work around. The only alternate proposal is to have something like

```
.. option:: -tilex TILEX
.. option:: -tiley TILEY
...
```
Leaving the decision up to you before pushing this.